backend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10-slim
```

Comment:
I get an error that `gcc` is not available. It works fine when switching to `python:3.10`, which should be fine if this is intended for development use. Otherwise installing `gcc` via apt-get below also works.